src/style/main/components/SensorDetails.scss

```diff
@@ -179,3 +190,32 @@
         }
     }
 }
+
+@media only screen and (max-width: 600px) {
```

Comment:
[Suggest] Do we trade with specific screen resolution? If yes, maybe we can specify standard screen resolution points in scss variables? 
I'm not sure what I said is a good idea. It's just suggestion.

Reply:
Good idea!